spec/bitbot/configuration_spec.rb

```diff
@@ -50,6 +50,15 @@ def self.type_name
     expect(subject.listeners[:custom_type]).to eq(config)
   end
 
+  context 'groups and whitelisting' do
+    it 'uses a default list of channels per group' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.